aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -1,4 +1,10 @@
 module aptos_framework::transaction_context {
+    /// Return the transaction hash of the current transaction
+    public native fun get_txn_hash(): vector<u8>;
+
+    /// Return a universally unique identifier
```

Comment:
Can you add more documentation here that this uuid is generated based on the txn hash and that multiple uuids can be created in the same tx

Reply:
Done.